apps/x509.c

```diff
@@ -785,6 +798,10 @@ int x509_main(int argc, char **argv)
                 goto err;
             }
         }
+    } else if (bundle) {
+        certs = sk_X509_new_null();
+        if (!load_certs(infile, 1, &certs, passin, NULL))
+            goto end;
```

Comment:
Move this into the next block after the `infile` check
because the `infile == NULL` case will be possible too, with basically the same warning,
where `Reading certificate` -> `Reading certificate(s)`

Reply:
Agreed.